### DISCUSSION TITLE
SmoothGraphics is used to draw a circle, which is composed of many points. The circle is not smooth after being magnified, and the amount of data is very large.

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#8873] [Q&A]
Whether the circle can be rendered with shaders, which can ensure that the circle is more smooth
Examples are as follows
Is there a plan for the next version of PIXI to change the circle, arc or PIXI.LINE_ CAP.ROUND, render with shaders and keep the arc smooth
https://www.pixiplayground.com/#/edit/-9ozT4cSsTjuyGauUlTIf
![image](https://user-images.githubusercontent.com/89827437/202973648-6b907939-cb40-46ca-ac7d-0d87ecaf025c.png)

**Comment by SuperSodaSea** [comment]
The URL you gave doesn't seem to match your description. Did you forgot to click Save in the playground?

**Comment by bigtimebuddy** [comment]
Smooth Graphic is a plugin. Would suggest you file an issue there:

https://github.com/pixijs/graphics-smooth
